### ISSUE TITLE
Unhelpful link

### ISSUE THREAD

**Issue by CaitM8Codes** [davorg/planetskeptic#8]
Hey, under sources the School Of Doubt links to a blank website. It would be good to have this lead to the book (e.g. https://www.google.com.au/books/edition/The_School_of_Doubt/B8KODwAAQBAJ?hl=en&gbpv=0) assuming that is the correct source. Unsure how to do this without changing href in index.html, but am trying to participate in Hacktober so would love to help! Cheers :)

**Comment by davorg** [comment]
Thanks for the report. It looks like School of Doubt was a blog I enjoyed, so I added its web feed to Planet Skeptic some years ago. But at some point since then the blog has ceased to exist - so it should be removed from the site.

I'd fix it myself, but as you pointed out the problem, I'm happy for you to do it and get the credit for Hacktoberfest. The README explains for to do that:

> `perlanetrc` contains YAML with the data that drives the file. Most importantly, it has the `feeds` list which defines the feeds that are aggregated into the web site (and a new Atom feed).

So to remove it, you just need to edit [`perlanetrc`](https://github.com/davorg/planetskeptic/blob/master/perlanetrc) and remove the section about School of Doubt (that's lines 43-45).
